apm-agent-plugins/apm-profiling-plugin/src/test/java/co/elastic/apm/agent/profiler/CallTreeTest.java

```diff
@@ -611,6 +612,46 @@ void testActivationAfterMethodEnds_RootChangesToB() throws Exception {
         });
     }
 
+    /*
+     * [1       ]
+     *  [a]
+     *     [2  ]
+     *      [b]
+     */
+    @Test
+    void testActivationAfterMethodEnds_RootChangesToB2() throws Exception {
+        assertCallTree(new String[]{
+            " aaa bbb  ",
+            "1   2   21"
+        }, new Object[][] {
+            {"a",     3},
+            {"b",     3},
+        }, new Object[][] {
+            {"1",     9},
+            {"  a",   2},
+            {"  2",   4},
+            {"    b", 2}
+        });
+    }
+
+
+    /*
+     * [1]
+     *  [a]
+    @Test
+    void testActivationBeforeCallTree() throws Exception {
+        assertCallTree(new String[]{
+            " aaa",
+            "1 1 "
+        }, new Object[][] {
+            {"a",   3},
+        }, new Object[][] {
+            {"a",   3},
+            {"  1", 2},
+        });
+    }     */
+
```

Comment:
What about this one? Do you want to keep it as something to fix in the future?

Reply:
Yes, I'll leave it in for that reason. Ideally, it should have a comment about what's not working and why but I don't remember the context.